src/Method.php

```diff
@@ -75,4 +73,16 @@ public function getUnavailableMessage(): string
     {
         return _t(__CLASS__ . '.REQUIRES_HTTPS', 'This method can only be used over HTTPS.');
     }
+
+    /**
+     * Get directory name this module is installed into
+     * SS3 is not specific or caring about what the module is named, only that it exists and is a valid SilverStripe
+     * mdoule.
+     *
+     * @return string
+     */
+    private function getModuleName(): string
+    {
+        str_replace(Director::baseFolder(), '', basename(__DIR__));
+    }
```

Comment:
I don't see any benefit in adding this TBH - ideally we'd keep the SS3 and SS4 versions as close as possible aside from the necessary API changes for compatibility with it. If this is to prevent writing "security-extensions" twice (which was already there) I don't think it's worthwhile